src/content/tutorials/continuous-deployment/apps/add_appcr.md

```diff
@@ -0,0 +1,195 @@
+---
+linkTitle: Add a new App to a workload cluster
+title: Add a new App to a workload cluster
+description: Learn how to deploy and configure applications into workload clusters using GitOps.
+weight: 90
+aliases:
+  - /advanced/gitops/apps
+menu:
+  principal:
+    identifier: tutorials-continuous-deployment-apps-wc
+    parent: tutorials-continuous-deployment-apps
+user_questions:
+  - How can I add an app to a workload cluster with GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+last_review_date: 2024-10-21
+---
+
+This document is part of the documentation to use GitOps with Giant Swarm app platform. You can find more information about the [app platform in our docs]({{< relref "overview/fleet-management/app-management/" >}}).
+
+# Add a new app to a workload cluster
+
+Follow the instructions below to add a new `App` to a workload cluster managed in this repository. You can add an `App` directly (without any intermediate step) or use an [app template]({{< relref "/tutorials/continuous-deployment/apps/add_app_template/" >}}). The documentation below shows common steps as well as what's different in both cases.
+
+## Examples
+
+Examples of creating apps are available in following locations:
+
+- An example of a directly configured app (the simplest use case - no configuration): an [app without configuration](https://github.com/giantswarm/gitops-template/tree/main//management-clusters/MC_NAME/organizations/ORG_NAME/workload-clusters/WC_NAME/apps/hello-world/)
+- An example of a directly configured app (with configuration): an [app that uses a configuration configmap](https://github.com/giantswarm/gitops-template/tree/main//management-clusters/MC_NAME/organizations/ORG_NAME/workload-clusters/WC_NAME/apps/ingress-nginx/)
+- An example of an app created from an app template is available in [WC_NAME/apps/ingress-nginx-from-template](https://github.com/giantswarm/gitops-template/tree/main//management-clusters/MC_NAME/organizations/ORG_NAME/workload-clusters/WC_NAME/apps/ingress-nginx-from-template/).
```

Comment:
those three links currently lead to a 404, maybe they can be adjusted with this PR?

Reply:
yeah thanks for the hint